content/CityFurniture/newrelic-deployment/index.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2019-09-19T00:00:00+00:00
+title: Newrelic Deployment
+author: CityFurniture
+tags: [ newrelic ]
+logo: newrelic.svg
+repo: CityFurniture/newrelic-deployment
+image: CityFurniture/newrelic-deployment
```

Comment:
Images are always lowercase, uppercase values are invalid at docker hub.

Reply:
And your examples clearly show that you are referencing the wrong image here.

Reply:
Should be fixed now, making a PR